Client/Application/AppLaunchUtil.swift

```diff
@@ -77,6 +77,7 @@ class AppLaunchUtil {
 
         RustFirefoxAccounts.startup(prefs: profile.prefs).uponQueue(.main) { _ in
             self.logger.log("RustFirefoxAccounts started", level: .info, category: .sync)
+            self.profile.pollCommands(forcePoll: false)
```

Comment:
Question about this to understand; if we have the `pollCommands` in `applicationDidBecomeActive`, it's necessary to have this one here because the `applicationDidBecomeActive` could be called before the RustFirefoxAccounts is startup?

Reply:
No this is a great catch! the `pollCommands` will only run the actual polling once the `RustFirefoxAccounts` has started up. i.e this was redundant - removing it!